release-testing/kv/client.go

```diff
@@ -0,0 +1,64 @@
+package kv
+
+import (
+	"fmt"
+	"net/rpc"
+	"os"
+)
+
+type StoreClient struct {
```

Comment:
Not a show-stopper, but the exported Golang types and functions will need comments... otherwise golint will complain.